releases/release-1.21/release-notes/maps/pr-92817-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 92817
+releasenote:
+  do_not_publish: true
```

Comment:
Yes! The note was removed from the draft successfully, but it is in the json file. Another one to fix before 1.21